app/views/notes/_submission_form.html.erb

```diff
@@ -0,0 +1,7 @@
+<%= form_for @note, url: project_notes_path, method: :post do |form| %>
+  <%= form.govuk_error_summary %>
```

Comment:
I agree that this isn't the right place for this, but I think ship it, note it in a card and we can prioritise fixing it around everything else as necessary.